storm-client/src/jvm/org/apache/storm/metrics2/reporters/ScheduledStormReporter.java

```diff
@@ -37,6 +37,10 @@ public static long getReportPeriod(Map<String, Object> reporterConf) {
         return ObjectReader.getInt(reporterConf.get(REPORT_PERIOD), 10).longValue();
     }
 
+    public static boolean getReportDimensions(Map<String, Object> reporterConf) {
```

Comment:
I would suggest to change this to `getReportDimensionsEnabled` or `isReportDimensionsEnabled`